example_simulation/example.conf

```diff
@@ -23,7 +23,7 @@ ensemble = [NVT]
 #         many then a list.
 potential = [openkim]
 # Arg:    Mandatory
-# Type:   String or list
+# Type:   String or list of strings
 # Discrp: Type of potential to be used on the materials. 

```

Comment:
"Description" should probably be shortened to "Desc".

Reply:
done